Spook/Oracles/Pricer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Phantasma.Domain;
+
+namespace Phantasma.Spook.Oracles
+{
+    public static class Pricer
+    {
+        public static decimal GetCoinRate(string baseSymbol, string quoteSymbol, string cryptoCompApiKey, bool cgEnabled, PricerSupportedToken[] supportedTokens)
+        {
+            try
+            {
+                Decimal cGeckoPrice = 0;
+                Decimal cCompare = 0;
+
+                cCompare = CryptoCompareUtils.GetCoinRate(baseSymbol, quoteSymbol, cryptoCompApiKey, supportedTokens);
```

Comment:
Nitpicking here, but maybe you call it `cComparePrice`  if the other price is called `cGeckoPrice`